### DISCUSSION TITLE
Mobile camera capture with 'capture' attribute for mv-popup img input

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#895] [Q&A] [answered]
Hi

Is there a way to support mobile camera capture to update an image inside <img> element?

reading here: [https://w3c.github.io/html-media-capture/](https://w3c.github.io/html-media-capture/)

Seems it just needs the 'capture' attribute on the input used in mv-popup

I tried to do something with a mv-editor="#custom", but I'm re-inventing the wheel, as mavo already manages all uploads, paths etc.. in this mv-popup

**Comment by LeaVerou** [comment]
Yeah there's no way to do this with current Mavo. But perhaps we could add it by default, I don't see why you may want to *not* have it. Thoughts, @karger @DmitrySharabin ?

**Comment by joyously** [comment]
Has the interface improved? When I was doing this on my web site a few years ago, putting the `accepts` or `capture` attribute made it work for some browsers, but made it fail for others. 
I had to use a plain file input to satisfy everyone.
I don't remember if `capture` forces the camera, or if you can choose a file already existing.

**Reply by GalinhaLX** [reply]
According to this [article](https://honeywellaidc.force.com/supportppr/s/article/How-to-use-HTML5-to-capture-a-camera-image)

"it depends on the browser engine. 
Some may directly offer to take a camera image and others may ask about an image source providing a list of possible image sources including the camera. 
Some device will not even support the HTML5 standard"

**Comment by GalinhaLX** [comment, marked as answer]
Found a way to do it without modifying mavo

[codepen here](https://codepen.io/galinhalx/pen/oNqwegm)


including it inside the initial `<img>` 'accept' attribute like this:

`accept="image/*;capture=camera"` 

=> works without Mavo modification
=> in my 'old' phone, under Chrome, proposes me to upload from file, or use camera
=> Compatible with 'cropper' (...except 'cropper' is not mobile friendly at all :(  even if I use some kind of scale transformation on mobile)

By the way, I'm interested to have your feedbacks, to know how it works on different devices
